src/Raven.Client/Documents/Subscriptions/SubscriptionBatch.cs

```diff
@@ -244,7 +249,7 @@ internal string Initialize(BatchFromServer batch)
                     {
                         try
                         {
-                            instance = _requestExecutor.Conventions.Serialization.DefaultConverter.FromBlittable<T>(curDoc, id);
+                            instance = _requestExecutor.Conventions.Serialization.SubscriptionsConverter.FromBlittable<T>(curDoc, id);
```

Comment:
This is using the same `SubscriptionsConverter` for all subscriptions which has a state.
This should be created per batch or per subscription.
Also need a test for this.